zebra-chain/src/types/amount.rs

```diff
@@ -21,6 +21,11 @@ impl<C> Amount<C> {
     {
         self.0.try_into()
     }
+
+    /// To little endian byte array
+    pub fn to_bytes(&self) -> [u8; 8] {
+        self.0.to_le_bytes()
+    }
```

Comment:
Hmm, this encodes an endianness assumption into the Amount type, but everywhere else that we serialize integers, we do so using the `byteorder` crate to make the endianness explicit at the point of serialization.  So my preference would be to continue that pattern rather than special-casing Amount.